build.assets/Makefile

```diff
@@ -139,7 +138,10 @@ buildbox:
 		--build-arg NODE_GRPC_TOOLS_VERSION=$(NODE_GRPC_TOOLS_VERSION) \
 		--build-arg NODE_PROTOC_TS_VERSION=$(NODE_PROTOC_TS_VERSION) \
 		--build-arg PROTOC_VER=$(PROTOC_VER) \
+		--build-arg PROTOC_VERSION=$(PROTOC_VERSION) \
```

Comment:
What's the difference between `PROTOC_VER` and `PROTOC_VERSION`?

Reply:
oh, thanks for spotting that. I thought I took are of it - on v13 its called PROTOC_VER. On v14+ it is PROTOC_VERSION.